ndex2/client.py

```diff
@@ -992,6 +1417,21 @@ def get_user_network_summaries(self, username, offset=0, limit=1000):
         It returns not only the networks that the user owns but also the networks that are
         shared with them directly.
 
+	.. code-block:: python
+	
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		username = "ENTER YOUR USER"
+		#offset is how far you start looking for user network summaries
+		#limit is the most networks you want to see
+
+		#the following code shows the summaries of the second to last and the third to last networks in my account 
+		my_ndex.get_user_network_summaries(username, offset=1, limit=2)
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client